lightkurve/correctors/sffcorrector.py

```diff
@@ -125,8 +125,10 @@ def correct(self, centroid_col=None, centroid_row=None, windows=20, bins=5,
                           LightkurveWarning)
 
         if centroid_col is None:
+            self.lc = self.lc[~np.isnan(self.lc.centroid_col)]
```

Comment:
Just to make this slightly more elegant, I propose changing it into this:
```suggestion
            self.lc = self.lc.remove_nans(column='centroid_col')
```